.eslintignore

```diff
@@ -1 +0,0 @@
-analytics.js
```

Comment:
Removed this file because 

- `.eslintignore` is no longer supported (see [here](https://github.com/MozillaFoundation/foundation.mozilla.org/actions/runs/10781425361/job/29899314881?pr=12833#step:5:26))
- and we no longer have `analytics.js` in the repo (this file was created 7 years ago)